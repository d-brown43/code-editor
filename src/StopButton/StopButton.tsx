import * as React from "react";

type StopButton = {
    stop: () => void;
}

const StopButton = ({stop}: StopButton) => {
    return (
        <button
            type='button'
            className='btn btn-outline-light'
            onClick={stop}
        >
            Stop
        </button>
    )
};

export default StopButton;
