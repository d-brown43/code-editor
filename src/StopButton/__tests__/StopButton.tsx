import StopButton from "../StopButton";
import * as React from "react";
import {shallow} from "enzyme";

it('matches snapshot', () => {
    const stopButton = shallow(
        <StopButton stop={jest.fn()} />
    );
    expect(stopButton).toMatchSnapshot();
});
