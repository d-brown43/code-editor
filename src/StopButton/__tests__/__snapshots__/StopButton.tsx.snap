// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<button
  className="btn btn-outline-light"
  onClick={[MockFunction]}
  type="button"
>
  Stop
</button>
`;
